explainers/update-rendering.md

```diff
@@ -68,6 +68,47 @@ rendering state is not updated to save CPU time. Adding `renderPriority` on
 such an element, however, would cause the User Agent to continually process its
 rendering with a given priority.
 
+### Examples
+
+Consider the following example.
+
+```html
+<div id=container style="content-visibility: hidden">
+  <!-- some complicated subtree here -->
+  ...
+</div>
+```
+
+Here the contents of the `#container` are not visible, and its rendering state
+is not necessarily up-to-date since the contents of the element are
+[skipped](https://www.w3.org/TR/css-contain-2/#skips-its-contents)
+
+This is typically done to avoid rendering work in this subtree. However, when
+the developer decides that the contents should now be visible, they remove
+`content-visibility: hidden` style. This causes all of the rendering to be
+updated in the subsequent frame. This work, in turn, can cause undue delay
+and visible jank on the page.
+
+The solution is to add the `renderPriority` attribute:
+
+```html
+<div id=container renderPriority=background style="content-visibility: hidden">
+  <!-- some complicated subtree here -->
+  ...
+</div>
+```
+
+Here, the developer has added `renderPriority=background`, which means that they
+would like the User Agent to keep the rendering state of `#container`'s content
+to be kept up to date with a low priority.
+
+Assuming that the User Agent completes this work (i.e. it is not otherwise busy
+doing more important rendering work), then when the developer removes
+`content-visibility` style, the contents are displayed without delay or jank.
```

Comment:
```suggestion
the `content-visibility` style, the contents are displayed without delay or jank.
```

Reply:
Add a caveat that this depends on the degree to which removing `content-visibility` causes a change of layout.